docs/self-managed/identity/user-guide/authorizations/generating-m2m-tokens.md

```diff
@@ -5,11 +5,15 @@ sidebar_label: "Generating machine-to-machine (M2M) tokens"
 description: "In this guide, we'll show you how to generate your own machine-to-machine (M2M) tokens."
 ---
 
-In this guide, we'll show you how to generate your own **machine-to-machine (M2M)** tokens.
+A **machine-to-machine (M2M)** token is a token requested by one service so it can
+communicate with another service acting as itself.
+
+In [Identity](/self-managed/identity/what-is-identity.md), we provide the ability to assign permissions to
+an application. This functionality allows an application to perform the `client_credentials` flow to
```

Comment:
🔧 This functionality doesn't strictly allow the `client_credentials` flow but rather another setting on the application.

 I wonder if we could say something like
```
... we provide the ability to assign permissions to an application. This allows an application to perform requests to other applications without the need for user intervention.
```

Reply:
Adjusted in latest commit 👍